src/neo-vm/OpCode.cs

```diff
@@ -327,6 +327,10 @@ public enum OpCode : byte
         /// </summary>
         PUSHM1 = 0x4F,
         /// <summary>
+        /// The item null is pushed onto the stack.
+        /// </summary>
+        PUSHNULL = 0x50,
```

Comment:
This is also not a good opcode location... 0x50 is supposed to be a `PUSH0`, but not created due to existing similar operation as `PUSHBYTES0`. Better not use 0x50 for this, and find another place (Null is not zero).

Reply:
`PUSH0` is `0x00`.

Reply:
PUSH0 pushes a ByteArray stack item with 0 bytes. I mean,this should be reserved for an opcode that pushes Integer stack item, with value 0,since this simplifies logic and understanding of the vm opcodes (my opinion).
But if you think that's fine as is, I wont argue.